docs/RationaleFIRRTL.md

```diff
@@ -59,12 +59,55 @@ community that would benefit from adding support for these, we can do so.
 
 ## Naming
 
-One of the goals of the FIRRTL compiler is to produce human-readable Verilog
-which can be easily mapped back to the input FIRRTL. Part of this effort
-means that we want the naming of Verilog objects to match the names used in
-the original FIRRTL, and they are predictably transformed during lowering.
-For example, after bundles are replaced with scalars in the lower-types pass,
-each field should be prefixed with the bundle name:
+Names in verilog form part of the public-api of a design.  It is therefore
+critical that Chisel, and by extension FIRRTL, have a language-level semantic 
+about how entities are named and how named entities are used and transformed.  
+This must specify which entities with names in Chisel generate predictable 
+output.  Since names server multiple purposes in a design; for example, 
+debugging, test-bench attachment, hooks for physical layout, etc; we must 
```

Comment:
Not sure about use of semicolons for bracketing what's usually a parenthetical (or with some sort of dashes around).  But works for me.

Reply:
Agree, I'd just use `(e.g., a, b, or c)`.